R/rlistings_methods.R

```diff
@@ -192,7 +198,6 @@ setMethod(
 )
 
 #' @rdname listing_methods
-#' @param obj The object.
```

Comment:
This was the culprit that caused the error message @Nolan-Steed

Reply:
Thanks @Melkiades for your help with this!